nBTC/sources/nbtc.move

```diff
@@ -287,7 +288,7 @@ fun verify_deposit(
     // see mint function for information about payload argument.
     _payload: vector<u8>,
     ops_arg: u32,
-): (u64, address) {
+): (u64, address, u64) {
```

Comment:
We can have multiple UTXO with the same spending_key. utxo_idx should be vector I think

Reply:
Ideally, there is only one. We need to think if we should support more than one. Having more is additional cost, and a user will need to pay more 
Let's add TODO for now.